docker/zms/conf/zms.properties

```diff
@@ -9,7 +9,7 @@
 
 # Comma separated list of authority implementation classes to support
 # authenticating principals in ZMS
-athenz.zms.authority_classes=com.yahoo.athenz.auth.impl.PrincipalAuthority,com.yahoo.athenz.auth.impl.UserAuthority,com.yahoo.athenz.auth.oauth.OAuthCertBoundJwtAccessTokenAuthority,com.yahoo.athenz.auth.impl.CertificateAuthority
+athenz.zms.authority_classes=com.yahoo.athenz.auth.impl.PrincipalAuthority,com.yahoo.athenz.auth.impl.TestUserAuthority,com.yahoo.athenz.auth.oauth.OAuthCertBoundJwtAccessTokenAuthority,com.yahoo.athenz.auth.impl.CertificateAuthority
```

Comment:
if this is a test image why have the other authorities? this might mislead others and use them as is and left behind the TestUserAuthority. If it's for local dev, then I expect TestUserAuthority should be the only one added.